processor/build.gradle

```diff
@@ -9,6 +9,9 @@ dependencies {
     implementation "net.openhft:zero-allocation-hashing:0.16"
     implementation "org.slf4j:slf4j-api:$slf4jVersion"
 
+    annotationProcessor "com.github.therapi:therapi-runtime-javadoc-scribe:$javaDocExtractorVersion"
+    compileOnly "com.github.therapi:therapi-runtime-javadoc:$javaDocExtractorVersion"
```

Comment:
https://github.com/dnault/therapi-runtime-javadoc

An annotation processor is required to retrieve the JavaDoc of `ProcessorProperties.PROPERTY_DEFINITIONS`.